uber/templates/base.html

```diff
@@ -53,7 +53,7 @@
                 {People: [
                     {Attendees: '../registration/'},
                     {Groups: '../groups/'},
-                    {'All Untaken Shifts': ''},
+                    {'All Untaken Shifts': '../jobs/everywhere'},
```

Comment:
The point of the page was to pull it up on-site to assign volunteers who come into Stops asking what's available, so it does things like only show shifts which haven't started yet and bypasses the usual restrictions on not being allowed to assign someone to a shift when they're not assigned to that department.

With that being said, it probably doesn't hurt to have the link, so I'll just go ahead and merge this PR now, and then we can always implement one of the other options later if that's better.